### PR TITLE
Pull request for Does my coding partner follow JS best practices?

### PR DESCRIPTION
**Pull request by Hamayounali**
Copied all file from todolist project

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <title>To Do List</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script><script defer src="print.bundle.js"></script></head>
+<body>
+    <br>
+    <div class="container">
+    <div><h3> Today's To Do</h3>
+        <form action="">
+            <div class="input-icons">
+            <button id="newBook" class="btn"><i class="fa fa-save"></i></button>
```
Comment (henrykgggg) [inline dist/index.html:15]: - [ ] please set attribute to your button

File: src/data.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    html += `<td class="test"><button class="btn" onclick="editTodo(${i}, '${todo.todoArr[i].description}')"> <i class="fa fa-edit" style="font-size:20px"></i> </button> </td>`;
+    html += `<td class="test"><button onclick="deleteTodo(${i})" class="btn"> <i class="fa fa-trash" style="font-size:20px"></i> </button> </td>`;
+    html += '</tr>';
+  }
+  html += '<tr>';
+  html += '<td colspan="4" class="clearButton" onclick="clearCompleted()"> <b> Clear all completed </b> </td>';
+  html += '</tr>';
+  html += '</table>';
+  document.getElementById('data').innerHTML = html;
+
```
Comment (henrykgggg) [inline src/data.js:28]: - [x] please  remove spaces in your code to make it clean

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// import _ from 'lodash';
```
Comment (henrykgggg) [inline src/index.js:1]: - [ ]  You should remove the unused codes/commented code in your codes so that your project looks cleaner.

**Review by henrykgggg — CHANGES_REQUESTED** [review]
Hi @Hamayounali  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing 👍 
PR is well described 👍 
Project is running successfully 👍 

_Check the comments under the review._

File: dist/index.html (hunk 1 above)
Comment (Hamayounali) [reply dist/index.html:15]: Ok, Thanks @henrykgggg  noted and changed accordingly

File: src/index.js (hunk 3 above)
Comment (Hamayounali) [reply src/index.js:1]: Thanks, Removed the unused line of code.

**Review by henrykgggg — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Hamayounali 

Your project is complete! Great job for  making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**